Doc/library/sqlite3.rst

```diff
@@ -1609,7 +1608,10 @@ Row objects
    It supports iteration, equality testing, :func:`len`,
    and :term:`mapping` access by column name and index.
 
-   Two row objects compare equal if have equal columns and equal members.
+   Two :class:`!Row` objects compare equal
+   if they have identical columns and values.
```

Comment:
If the `Row` objects were from different DBs that happened to have the same selected column names, would they compare equal? If so, then would

```suggestion
   if they have identical column names and values.
```

be more accurate?

Reply:
Different databases does not matter; column names are compared first, then the tuples with the raw row results. So, yeah, this is more accurate. Thanks!